app/models/shop.rb

```diff
@@ -0,0 +1,4 @@
+class Shop < ApplicationRecord
+  belongs_to :user
+  has_many :shop_participants
```

Comment:
sorry guys quick question - what's the logic behind LINE 3?

A single shop belongs to a user(chairperson) and this same single shop CAN PARTICIPATE in MANY stamp rallies?

Reply:
Yes, I think that's it! Since shop_participants is the join table for shops and the relation is one to many, one shop can participate in several stamp_rallies, so one shop has, at some point, many shop_participants...

Reply:
Now that I think about it, maybe we don't need this association 🤔

Reply:
ok thanks that helps - i think i was mixing the two join tables

but i think we do need the association because that's how we're going to track participants THROUGH the shops they visit when they scan the code....?